### PR TITLE
Some words are missing in Malayalam in Judge login

### PR DESCRIPTION
**Pull request by utpalBeehyv**
https://github.com/pucardotorg/dristi/issues/2281

## Release Notes

- **New Features**
  - Enhanced localization for various components, including dynamic translations for button labels and UI text.
  - Improved delivery channel representation with translatable labels in the Summons Order component.

- **Bug Fixes**
  - Maintained error handling for mandatory fields across components to ensure user prompts are displayed correctly.

- **Chores**
  - Updated configuration labels for clarity and consistency across multiple components, enhancing user interface readability.

### CODE REVIEW

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/admission/ScheduleAdmission.js [hunk 1]
```diff
@@ -187,7 +187,7 @@ function ScheduleAdmission({
             modalInfo?.showCustomDate ? setModalInfo({ ...modalInfo, page: 0, showDate: false, showCustomDate: false }) : setShowModal(false);
           }}
           className="primary-label-btn back-from-schedule"
-          label={"Back"}
+          label={t("DOWNLOAD_CS_BACK")}
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/admission/ScheduleAdmission.js:190]: Irrelavant

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/FSOHomeConfig.js [hunk 2]
```diff
@@ -356,25 +356,25 @@ export const TabFSOSearchConfig = {
           uiConfig: {
             columns: [
               {
-                label: "Case Name",
+                label: "CS_CASE_NAME",
                 jsonPath: "caseTitle",
               },
               {
-                label: "Scrutiny Status",
+                label: "CS_SCUTINY_STATUS",
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/FSOHomeConfig.js:363]: Done

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/FSOHomeConfig.js [hunk 3]
```diff
@@ -208,24 +208,24 @@ export const TabFSOSearchConfig = {
...
               },
               {
-                label: "Case Type",
+                label: "CASE_TYPE",
                 jsonPath: "",
                 additionalCustomization: true,
               },
               {
-                label: "Days Since Filing",
+                label: "CS_DAYS_FILING",
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/FSOHomeConfig.js:228]: Irrelavant

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/CourtRoomHomeConfig.js [hunk 4]
```diff
@@ -146,7 +146,7 @@ export const TabCourtRoomSearchConfig = {
             defaultValues: defaultSearchValues,
             fields: [
               {
-                label: "Case Type",
+                label: "CASE_TYPE",
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/CourtRoomHomeConfig.js:149]: Irrelavant

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/JudgeHomeConfig.js [hunk 5]
```diff
@@ -103,7 +103,7 @@ export const TabJudgeSearchConfig = {
   showTab: true,
   TabSearchConfig: [
     {
-      label: "All",
+      label: "CS_ALL",
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/JudgeHomeConfig.js:106]: No Need

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/components/TaskComponent.js [hunk 6]
```diff
@@ -522,7 +522,9 @@ const TasksComponent = ({
             <React.Fragment>
               <div className="task-filters">
                 <LabelFieldPair>
-                  <CardLabel style={{ fontSize: "16px" }} className={"card-label"}>{`Case Type`}</CardLabel>
+                  <CardLabel style={{ fontSize: "16px" }} className={"card-label"}>
+                    {t("CASE_TYPE")}
+                  </CardLabel>
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/components/TaskComponent.js:527]: irrelevant

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/JudgeHomeConfig.js [hunk 7]
```diff
@@ -205,24 +205,24 @@ export const TabJudgeSearchConfig = {
           uiConfig: {
             columns: [
               {
-                label: "Case Name",
+                label: "CS_CASE_NAME",
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/JudgeHomeConfig.js:208]: irrelevant

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/JudgeHomeConfig.js [hunk 8]
```diff
@@ -142,7 +142,7 @@ export const TabJudgeSearchConfig = {
             defaultValues: defaultSearchValues,
             fields: [
               {
-                label: "Case Type",
+                label: "CASE_TYPE",
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/JudgeHomeConfig.js:145]: irrelevant

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/FSOHomeConfig.js [hunk 9]
```diff
@@ -95,7 +95,7 @@ export const TabFSOSearchConfig = {
   showTab: true,
   TabSearchConfig: [
     {
-      label: "All",
+      label: "CS_ALL",
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/FSOHomeConfig.js:98]: irrelevant

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/UICustomizations.js [hunk 10]
```diff
@@ -385,13 +385,13 @@ export const UICustomizations = {
     },
     additionalCustomizations: (row, key, column, value, t, searchResult) => {
       switch (key) {
-        case "Case Type":
+        case "CASE_TYPE":
           return <span>NIA S138</span>;
-        case "Filing Date":
+        case "CS_FILING_DATE":
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/UICustomizations.js:390]: No need

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/UICustomizations.js [hunk 11]
```diff
@@ -298,11 +298,11 @@ export const UICustomizations = {
       const today = new Date();
       const formattedToday = new Date(today.getFullYear(), today.getMonth(), today.getDate());
       switch (key) {
-        case "Case Type":
+        case "CASE_TYPE":
           return <span>NIA S138</span>;
-        case "Scrutiny Status":
+        case "CS_SCRUTINY_STATUS":
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/UICustomizations.js:303]: irrelavant

**Review by vaibhavct — APPROVED** [review]

**Review by vaibhavct — APPROVED** [review]

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/orders/src/components/SummonsOrderComponent.js [hunk 12]
```diff
@@ -4,50 +4,55 @@ import { Button, Dropdown } from "@egovernments/digit-ui-react-components";
...
       <form>
-        {deliveryChannels.map((channel) => (
-          <div key={channel?.type}>
-            {Array.isArray(channel?.values) && channel?.values?.length > 0 && channel?.values[0] != null && (
-              <div>
-                <h2>
-                  <strong>{channel.type} to </strong>
-                </h2>
+        {deliveryChannels.map((channel) => {
+          console.log(channel);
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/orders/src/components/SummonsOrderComponent.js:21]: remove in next pr

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/UICustomizations.js [hunk 13]
```diff
@@ -385,13 +385,13 @@ export const UICustomizations = {
...
+        case "CASE_TYPE":
           return <span>NIA S138</span>;
-        case "Filing Date":
+        case "CS_FILING_DATE":
           return <span>{formatDate(new Date(value))}</span>;
-        case "Outcome":
+        case "CD_OUTCOME":
           return t(value);
-        case "Stage":
+        case "CS_STAGE":
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/UICustomizations.js:394]: No Need

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/orders/src/components/SummonsOrderComponent.js [hunk 14]
```diff
@@ -62,19 +67,21 @@ const SummonsOrderComponent = ({ t, config, formData, onSelect, clearErrors }) =
...
       values: [],
     },
     {
+      label: "REGISTERED_POST",
       type: "Registered Post",
       code: "RPAD",
       values: [],
     },
-    { type: "Via Police", code: "POLICE", values: [] },
+    { label: "VIA_POLICE", type: "Via Police", code: "POLICE", values: [] },
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/orders/src/components/SummonsOrderComponent.js:84]: No need

**Pull request merged**